src/commands/utils/config.ts

```diff
@@ -72,6 +73,24 @@ export function getKubectlGadgetConfig(): Errorable<KubeloginConfig> {
     return { succeeded: true, result: configresult };
 }
 
+export function getDraftConfig(): Errorable<DraftConfig> {
+    const draftConfig = vscode.workspace.getConfiguration('aks.drafttool');
+    const props = combine([getConfigValue(draftConfig, 'releaseTag')]);
```

Comment:
`combine` is not needed here - can just do:
```typescript
const props = getConfigValue(draftConfig, 'releaseTag');
```
And then line 88 would be `releaseTag: props.result`.